fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/HomeActivityTestRule.kt

```diff
@@ -31,6 +32,7 @@ class HomeActivityTestRule(
     initialTouchMode: Boolean = false,
     launchActivity: Boolean = true,
     private val skipOnboarding: Boolean = false,
+    private val hideKeyboard: Boolean = false,
```

Comment:
Just for consistency let's use `hideSoftKeyboard` to match the function name you defined later on.